cookbook/doctrine/registration_form.rst

```diff
@@ -45,7 +45,7 @@ You have a simple ``User`` entity mapped to the database::
         /**
          * @ORM\Column(type="string", length=255)
          * @Assert\NotBlank()
-         * @Assert\Length(max = 4096)
+         * @Assert\Length(max=4096)
```

Comment:
@xelaris to avoid getting an exception when trying to encode it (a malicious user should still receive a 400 status code rather than a 500)